Cargo.toml

```diff
@@ -14,16 +14,22 @@ categories = ["command-line-utilities"]
 # See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
 
 [dependencies]
-anyhow = "1.0.71"
 clap = { version = "4.3.0", optional = true, features = ["unicode", "derive"] }
+colored = "2"
```

Comment:
we like to use nu-ansi-term for ansi coloring

Reply:
I can make the change

Reply:
Done

> we like to use nu-ansi-term for ansi coloring

Can I ask why, though? Is nu-ansi-term only used in the nushell org or also outside?

Reply:
nu-ansi-term is forked by us and maintained by us. we like the ansi escape language it supports and is used in nushell and reedline.